examples/upsert-resource-multiple/README.md

```diff
@@ -0,0 +1,30 @@
+# upsert-resource: Upsert Resource Multiple Example
+
+In this example, we will see how to upsert multiple resources using `upsert-resource`
+function.
+
+### Fetch the example package
+
+Get the example package by running the following commands:
+
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/upsert-resource-multiple
+```
+
+kpt CLI accepts only one resource as fn-config. Hence, `upsert-resource` function 
+accepts Resource `List` which is used to wrap multiple resources to upsert. 
+You can find an example ofqq `List` at `.expected/fn-config.yaml`
+
+### Function invocation
+
+Invoke the function by running the following commands:
+
+```shell
+$ kpt fn eval -i upsert-resource:unstable --fn-config .expected/fn-config.yaml
```

Comment:
> .expected/fn-config.yaml

`.expected` is used to host test harness related stuff while `fn-config.yaml` is user-facing content. It should live in another directory.

Reply:
I feel fn-config is input to function and it should live next to `config.yaml` or `exec.sh`.